src/scripts/globalize.js

```diff
@@ -57,6 +57,12 @@ define(['connectionManager', 'userSettings', 'events'], function (connectionMana
             currentDateTimeCulture = currentCulture;
         }
         ensureTranslations(currentCulture);
+        // FIXME: See GH #1027 and #913. This should be configurable and not strictly based on locale.
+        if (currentCulture == 'zh-tw' || currentCulture == 'zh-hk') {
+            require(["css!jellyfin-noto/css/TC"]);
+        } else {
+            require(["css!jellyfin-noto/css/SC"]);
```

Comment:
I still have issues with it loading Simplified Chinese by default over Japanese, since they also share characters and we're not sure if characters are different or not.

As-is, this fixes Traditional Chinese and Simplified Chinese, but we're not sure if it'd still screw up Japanese text or not.

I maintain that, in my opinion, Japanese should be the only script of this kind loaded by default (Because of Hiragana and Katakana, that aren't shared with any other script) and either variant of Chinese should be loaded only in their respective case.

Reply:
@jellyfin/web This is, of course, open to discussion if anyone can reliably confirm that all shared characters of Simplified Chinese and Japanese are exactly identical.

Reply:
There are some difference between SC and JP. bt most people from east asian can read all the different writings. 
for example 
<span lang= ja>门</span> and <span lang=zh-cn>门</span> are the same code point. you can copy and paste to see they are the same.

Reply:
Thanks for the info :)

Then I'm positive we want to give people the best experience, so loading their relevant language is the best approach. We ideally want everyone to be a first-class citizen and, while they certainly are able to read variations, it's not much work to make sure it uses the proper ones, so might as well do it :)

Reply:
This means we'll have to make [jf-noto](https://github.com/jellyfin/jellyfin-noto) to _not_ include all CJK by default as well.

Reply:
This can be done with an additional check for Japanese. What's going to happen with those with other languages though?

Reply:
I think non-CJK languages don't have conflicting code points, so they're handled by the browser just fine.

Reply:
@JustAMan But I have items with Japanese text and my UI is in Spanish. How we should handle that?

Reply:
We should pick one variant as the default probably and make the logic do like
```
switch (locale) {
    case tradidional chinese:
          load TC.css
    case simplified chinese:
          load SC.css
// etc.
    default (no matching locale)
         load someDefault.css
```

Reply:
each CJK country select their own version of the same character. but they are the same character. most people who can read Han characters would understand theses variants. Noto CJK fonts also include latin fonts so displaying Spanish and Jp at the same time is not an issue. 
Html attribute "lang" and CSS selector lang() could be used to display these variant correctly. I did that with  <span lang=ja>门</span> and <span lang=zh-cn>门</span>. 
my own media library includes SC, TC, and JP contents. I can read most of the variants so it has never been an issue. But to make things perfect, there should be an option for users to specify the display language of the contents. Maybe this can be done with custom css file.

Reply:
What we thought about as a more permanent solution (in a further PR for 10.6) is to add a Display option to choose between Japanese, Traditional Chinese, and Simplified Chinese.

The default would be set to Japanese, which would then always be loaded (Due to hiragana and katakana) but would set itself automatically to the relevant option when a Chinese variant is selected. The user would also be free to override the setting to whatever they please afterward.

This seems to give the best support for multi-lingual libraries, while respecting local expectations for the look of various characters.

Reply:
Both hiragana and katakana should be included in any variants of noto sans CJK as the same. I checked windows Character Map on NotoSansCJKsc OTF. Hiragana and katakana are from U+3042 to U+30FF. But I have no idea how jellyfin-noto was subsetted.  The differences of NotoCJKsc, tc, jp are their Han characters.

Reply:
@frankyifei You can see how they are subsetted [here](https://github.com/jellyfin/jellyfin-noto/blob/master/font-faces.css).

As you can see, it's a replica of Google Fonts: we set in CSS a reference to each of the ranges a woff2 file support and the browser loads only the woff files that contains those characters, so we save bandwidth.

In [these lines](https://github.com/jellyfin/jellyfin-noto/blob/b784602db063734c721a46563ae5d6577ec2b35d/font-faces.css#L194-L213) you can see where the CJK fonts are loaded. [Before](https://github.com/jellyfin/jellyfin-noto/blob/9baaa18d15e644e682a9fc0fec9a5e81b3eb3caa/font-faces.css), all of them (JP, HK, KR, TC, SC) were loaded. Now, TC was removed and it's loaded by jellyfin-web once the client is set to traditional Chinese. So, by default, SC, HK, KR and JP are loaded, so users who have foreign languages but have items in CJK characters are covered.

Reply:
I looked the 5 .css files roughly.
Everyone of them includes hiragana and katakana.
Only KR includes hangual, other don't. 
Except KR, other .css share quite large amount of code points.

But it does not really matter because I just found out most of the text such as movie titles and descriptions are displayed in chrome are rendered with local fonts. Only a few of the ui text use webfonts.

Reply:
@frankyifei This is happening in Chrome Windows/Linux/Mac v80? Any sample text where the font isn't applied? Can you attach an screenshot?

Reply:
I guess it should work. but HK is not necessary.

Reply:
firefox, chrome on windows. I dont know why is this happenning.  Look the font style is different when I set local font to serif Noto CJK. 
![Screenshot_2020-04-10 Jellyfin(1)](https://user-images.githubusercontent.com/8104251/78953795-272ed180-7b19-11ea-93d1-083df87dec33.png)